charts/codesealer/templates/config-toml-configmap.yaml

```diff
@@ -44,6 +48,8 @@ data:
     maxFileSize = {{ .Values.worker.config.cache.maxFileSize }}
     maxSize = {{ .Values.worker.config.cache.maxSize }}
 
+    {{- if .Values.worker.config.bootloader.fsEndpoints }}
```

Comment:
I'm not sure the logic is sound here. `.Values.manager.enabled` and `.Values.worker.config.bootloader.fsEndpoints` should be mutually exclusive. The easiest way to do this is to set `fsEndpoints` in this config depending on those two values, since this entire section is ignored anyway if `fsEndpoints` is false. 

We can of course still omit this section based on the value of `fsEndpoints` we actually set, if we want to make the configmap simpler when the manager is enabled.

Reply:
Made changes to only use .Values.manager.enabled to enable standalone mode.  Let me know if you want to call this something different.